charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -28,6 +28,10 @@ spec:
               value: "{{- printf "%s-%s" .Release.Name "s3-metadata" | trunc 63 | trimSuffix "-" -}}"
             - name: REDIS_HOST
               value: {{ template "redis.fullname" . }}
+            - name: ENDPOINT
```

Comment:
I really don't get why this is required *when using Orbit*. Which value should one set this to? This feels more like a bug in Cloudserver rather than something to work-around here...

Reply:
Hmm we could not reproduce the error we were getting yesterday. However the original error in the logs were that the default "cloudserver-front" endpoint listed in the config.json wasn't reachable. 

I also confirmed with Rached that the ```ENDPOINT``` variable should NOT be necessary when using orbit. This commit will be deleted.